app/config/FrontendAppConfig.scala

```diff
@@ -48,6 +48,9 @@ class FrontendAppConfig @Inject()(config: Configuration, servicesConfig: Service
   lazy val lostUtrUrl : String = config.get[String]("urls.lostUtr")
   lazy val logoutUrl: String = config.get[String]("urls.logout")
 
+  val maintainATrustAgentDeclarationUrl: String = config.get[String]("urls.agentDeclaration")
```

Comment:
Make this redirect back to the maintain-a-trust-frontend service task list overview
@JoshuaH01